selectors/views/custom_status.ts

```diff
@@ -11,14 +11,18 @@ import {UserCustomStatus} from 'mattermost-redux/types/users';
 
 import {GlobalState} from 'types/store';
 
+// const timer = 1616155500110;
 export function makeGetCustomStatus(): (state: GlobalState, userID?: string) => UserCustomStatus {
```

Comment:
Use a normal selector `getCustomStatus` instead of using `makeGetCustomStatus` function that returns a selector